cmd/setup.go

```diff
@@ -115,6 +116,7 @@ type Setup struct {
 	Interactive   bool
 }
 
+//Run runs setup steps
```

Comment:
lol I love that comment!

Reply:
lol i just wanted to remove that blue underline warning from my vscode